app/src/main/java/org/fossasia/openevent/general/discount/DiscountApi.kt

```diff
@@ -7,7 +7,7 @@ import retrofit2.http.Path
 
 interface DiscountApi {
 
-    @GET("events/{id}/discount-code?include=event&fields[event]=id")
+    @GET("events/{id}/discount-codes?include=event&fields[event]=id")
     fun getDiscountCodes(@Path("id") id: Long): Single<DiscountCode>
```

Comment:
In the API it is stated that a list returns, are you sure that only a single discount code is present everytime ? as there might be cases where there are multi discount codes in some events right ?